src/main/java/com/celements/metatag/enums/twitter/ETwitterCardType.java

```diff
@@ -30,8 +29,8 @@ public String getIdentifier() {
   }
 
   @NotNull
-  public static Optional<ETwitterCardType> getTwitterCardType(@Nullable String identifier) {
-    return ID_MAP.get(identifier);
+  public static ETwitterCardType getTwitterCardType(@Nullable String identifier) {
```

Comment:
@msladek I will keep it in mind to check better, thank you that you did it, I missed this one